bundle/LegacyMapper/SiteAccess.php

```diff
@@ -47,6 +47,10 @@ public function onBuildKernelWebHandler(PreBuildKernelWebHandlerEvent $event)
         $request = $event->getRequest();
         $uriPart = array();
 
+        // @todo Remove when, if ever, legacy does not use deprecated constructor names
+        // triggers initial autoload of eZSiteAccess and sileces deprecation warning
+        @eZSiteAccess::TYPE_DEFAULT;
```

Comment:
We can, but for symfony 3.0 support we actually want to have deprecation errors make test fails on other things, hence the selective silencing when loading legacy classes.